### ISSUE TITLE
ENOATTR undeclared

### ISSUE THREAD

**Issue by edwintorok** [janestreet/core#122]
On Fedora 29 silverblue installing `core` v0.13-preview.120.22+294 fails with:
```
ocamlc src/linux_ext_stubs.o (exit 2)
(cd _build/default/src && /usr/bin/ocamlc.opt -g -I /usr/lib64/ocaml/threads -I /var/home/edwin/.opam/default/lib/base -I /var/home/edwin/.opam/default/lib/base/caml -I /var/home/edwin/.opam/default/lib/base/md5 -I /var/home/edwin/.opam/default/lib/base/shadow_stdlib -I /var/home/edwin/.opam/default/lib/base_bigstring -I /var/home/edwin/.opam/default/lib/base_quickcheck -I /var/home/edwin/.opam/default/lib/bin_prot -I /var/home/edwin/.opam/default/lib/bin_prot/shape -I /var/home/edwin/.opam/default/lib/core_kernel -I /var/home/edwin/.opam/default/lib/core_kernel/base_for_tests -I /var/home/edwin/.opam/default/lib/fieldslib -I /var/home/edwin/.opam/default/lib/jane-street-headers -I /var/home/edwin/.opam/default/lib/parsexp -I /var/home/edwin/.opam/default/lib/ppx_assert/runtime-lib -I /var/home/edwin/.opam/default/lib/ppx_bench/runtime-lib -I /var/home/edwin/.opam/default/lib/ppx_compare/runtime-lib -I /var/home/edwin/.opam/default/lib/ppx_enumerate/runtime-lib -I /var/home/edwin/.opam/default/lib/ppx_expect/collector -I /var/home/edwin/.opam/default/lib/ppx_expect/common -I /var/home/edwin/.opam/default/lib/ppx_expect/config -I /var/home/edwin/.opam/default/lib/ppx_hash/runtime-lib -I /var/home/edwin/.opam/default/lib/ppx_inline_test/config -I /var/home/edwin/.opam/default/lib/ppx_inline_test/runtime-lib -I /var/home/edwin/.opam/default/lib/ppx_module_timer/runtime -I /var/home/edwin/.opam/default/lib/ppx_sexp_conv/runtime-lib -I /var/home/edwin/.opam/default/lib/sexplib -I /var/home/edwin/.opam/default/lib/sexplib/unix -I /var/home/edwin/.opam/default/lib/sexplib0 -I /var/home/edwin/.opam/default/lib/spawn -I /var/home/edwin/.opam/default/lib/splittable_random -I /var/home/edwin/.opam/default/lib/stdio -I /var/home/edwin/.opam/default/lib/time_now -I /var/home/edwin/.opam/default/lib/typerep -I /var/home/edwin/.opam/default/lib/variantslib -ccopt -g -ccopt -D_LARGEFILE64_SOURCE -ccopt -I../../core_kernel/src -o linux_ext_stubs.o linux_ext_stubs.c)
linux_ext_stubs.c: In function ‘linux_getxattr’:
linux_ext_stubs.c:663:12: error: ‘ENOATTR’ undeclared (first use in this function); did you mean ‘ENOTTY’?
       case ENOATTR:
            ^~~~~~~
            ENOTTY
linux_ext_stubs.c:663:12: note: each undeclared identifier is reported only once for each function it appears in
linux_ext_stubs.c: In function ‘linux_setxattr’:
linux_ext_stubs.c:718:12: error: ‘ENOATTR’ undeclared (first use in this function); did you mean ‘ENOTTY’?
       case ENOATTR:
            ^~~~~~~
            ENOTTY
    ocamlopt src/.core.objs/native/core__Iobuf.{cmx,o}
File "src/iobuf.ml", line 1080, characters 4-62:
Warning 55: Cannot inline: Function information unavailable
```

**Comment by kevinji** [comment]
I think this issue and #123 can be closed because of 069bacd90d923bd22fe45bebb3097ec96a4d8a50.

**Comment by xclerc** [comment]
Indeed, it is fixed.
